### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by elmar8287**
- Add JS objects
- Add dynamic using of information stored in that JavaScript object
- Style according Figma template

### CODE REVIEW

**Comment by pasytchangwa** [comment]
Hello @elmar8287 , great job so far. I noticed that the border-right of your live portfolio version (mobile and desktop) is not as required in the original version. One thing you could do to remedy this is to remove the overflow: scroll added to the modal content in CSS as shown below. https://github.com/elmar8287/elmar8287.github.io/blob/9dff2a95ff0c1e4e1f26548306884351fa227168/index.css#L741-L742,

- You could as well check the alignment of your card items in the desktop version. They are currently not aligned as per the original requirement in Figma. A display 'flex' of the description and technologies will resolve that. Keep it up 

figma

**Comment by elmar8287** [comment]
Dear @pasytchangwa,

Thank you for the comment. I am going to fix these bugs.

My best regards,

Elmar

File: index.html [hunk 1]
```diff
@@ -48,9 +49,9 @@
...
       </nav>
     </header>
-    
+
     <section class="headline">
-      
+
       <h1 class="header-1 color-n800 mb-12">
         I’m Elmar,<br />Glad to see you!
       </h1>
```
Comment (smcommits) [inline index.html]: - [x] **Please make sure that the projects are created dynamically using Javascript.**

In this milestone, the already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. Please create objects with each object containing information about your projects. Then use that array to dynamically create projects.

File: index.html [hunk 2]
```diff
@@ -105,6 +106,46 @@ <h2 class="header-3 color-n800 mb-12">Tonic</h2>
         </ul>
         <div class="action">
           <button id="myBtn" type="button" class="button cardButton myBtn1">See Project</button>
+
+          <!-- modal -->
+          <div id="myModal" class="modal">
```
Comment (smcommits) [inline index.html]: - [x] **Please make sure that the Modal popup data is populated with the data from the projects array.**

Please use the projects array mentioned in the previous comment to populate data in your modals.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @elmar8287 , 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️

- [x] **Please add Eslint to your GitHub actions**

Eslint is not initialized in your pull request. Please make sure to set up Eslint with your Github actions. 

_Please check the inline comments below for other requested changes._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

File: index.html (hunk 2 above)
Comment (elmar8287) [reply index.html]: Dear @smcommits,

I have fixed it. Please, check.

MR

Elmar

File: index.html (hunk 1 above)
Comment (elmar8287) [reply index.html]: Dear @smcommits,

I have fixed it. Please, check.

MR

Elmar

File: index.js [hunk 3]
```diff
@@ -1,211 +1,197 @@
...
+const cardInfo = document.querySelectorAll('.info');
+const cardDesc = document.querySelectorAll('.section-info');
+const cardTech = document.querySelectorAll('.program-btns');
+const cardBtn = document.querySelectorAll('.myBtn');
+let card1 = {};
+let card2 = {};
+let card3 = {};
+let card4 = {};
+const cardObjects = [
+  (card1 = {
```
Comment (Temmarie) [inline index.js:13]: - [x] Your project is approved, but fix this before moving on to the next milestone.
- Remove the text that shows up on the popup window underneath the images.
_See image below._

image

**Review by Temmarie — APPROVED** [review]
### Status: Approved ✔️ 
#### Hi @elmar8287  , 
 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: index.js (hunk 3 above)
Comment (elmar8287) [reply index.js:13]: Dear @Temmarie,

Thank you for your approval. I will fix your comments.

My regards,

Elmar

**Pull request merged**